ui/src/components/process.vue

```diff
@@ -179,6 +185,10 @@ const async = require('async');
 //store full list of datatypes / projects names...
 let cache_datatypes = null;
 let cache_projects = null;
+function getHashValue(key) {
```

Comment:
Let's just set the t.id like following
> https://brainlife.io/project/5a022fc99c0d250055709e9c/process/5b2a4cba93c6a50048c7af09#4

#4 corresponds to t.4 inside the opened process.